src/components/ImageGeneration/GenerationForm/ResourceSelectMultiple.tsx

```diff
@@ -57,28 +58,45 @@ const ResourceSelectMultiple = forwardRef<HTMLDivElement, ResourceSelectMultiple
       if (_values.length !== value.length) onChange?.(_values.length ? _values : undefined);
     }, [value]); //eslint-disable-line
 
+    // Made with copilot :^) -Manuel
```

Comment:
Would it make more sense to sort the groups based on `options.types`? For example, in the constants file, we have `generationConfig.SD1.additionalResourceTypes = [ModelType.LORA, ModelType.TextualInversion, ModelType.LoCon]`. We could change the order of the types here to control the group sort.

`[ModelType.LORA, ModelType.LoCon, ModelType.TextualInversion]`